code/API_definitions/qod-api.yaml

```diff
@@ -347,18 +420,213 @@ components:
+
+            For 5G network the 3GPP specification TS 23.203 defines the packet error loss rate QCI attribute. It
+            describes the Quality of Service (QoS) Class Identifier (QCI) parameters used to
+            differentiate traffic classes in mobile networks, ensuring appropriate resource
+            allocation and performance for various services.
+
+            The packet error loss rate is one of the QCI attributes, providing information on the
+            acceptable packet loss rate for a specific traffic class. This attribute helps maintain
+            the desired performance level for services like voice calls, video streaming, or data
+            transfers within the 3GPP mobile network.
+          format: int32
+          minimum: 1
+          maximum: 10
+          example: 3
+      required:
+        - name
+        - status
+    QosProfileName:
+      description: |
+        A unique name for identifying a specific QoS profile.
+        This may follow different formats depending on the service providers implementation.
+        Some options addresses:
+          - A UUID style string
+          - Support for predefined profiles QOS_S, QOS_M, QOS_L, and QOS_E
+          - A searchable descriptive name
+      type: string
+      example: QCI_1_voice
+      minLength: 3
+      maxLength: 256
+      format: string
+      pattern: "^[a-zA-Z0-9_.-]+$"
+    Rate:
+      type: object
+      properties:
+        value:
+          type: integer
+          example: 10
+          format: int32
+          minimum: 0
+          maximum: 1024
+        unit:
+          $ref: "#/components/schemas/RateUnitEnum"
+    Duration:
+      type: object
+      properties:
+        value:
+          type: integer
+          example: 12
+          format: int32
+          minimum: 1
+        unit:
+          allOf:
+            - $ref: "#/components/schemas/TimeUnitEnum"
+            - example: Minutes
+    TimeUnitEnum:
       type: string
       enum:
-        - QOS_E
-        - QOS_S
-        - QOS_M
-        - QOS_L
+        - Days
+        - Hours
+        - Minutes
+        - Seconds
+        - Milliseconds
+        - Microseconds
+        - Nanoseconds
+    QosProfileStatusEnum:
       description: |
-        * `QOS_E` - Qualifier for enhanced communication profile
-        * `QOS_S` - Qualifier for the requested QoS profile _S_
-        * `QOS_M` - Qualifier for the requested QoS profile _M_
-        * `QOS_L` - Qualifier for the requested QoS profile _L_
-    
+        The current status of the QoS Profile
+        - `ACTIVE`- QoS Profile is available to be used
+        - `INACTIVE`- QoS Profile is not currently available to be deployed
+        - `DEPRECATED`- QoS profile is actively being used in a QoD session, but can not be deployed in new QoD sessions
+      type: string
+      enum:
+        - ACTIVE
+        - INACTIVE
+        - DEPRECATED
+    RateUnitEnum:
+      type: string
+      enum:
+        - bps
+        - Kbps
```

Comment:
This should be `kbps`. Symbol for "kilo" is "k". See [SI Prefixes](https://en.wikipedia.org/wiki/Metric_prefix).